react/global.css

```diff
@@ -1,51 +1,45 @@
-@media screen and (max-width: 40em) {
-  .vtex-modal__overlay {
-    align-items: stretch !important;
+@media screen and (min-width: 40em) {
+  .vtex-product-customizer__options {
+    background-color: transparent;
```

Comment:
Tachyons already provides a property for transparent backgrounds as you can see [here](https://tachyons.io/docs/themes/skins/) (search for `.bg-transparent`)